packages/near-membrane-base/src/membrane.ts

```diff
@@ -610,6 +643,34 @@ export function createMembraneMarshall() {
             return fn;
         }
 
+        function isLiveTarget(target: ProxyTarget): boolean {
+            if (target === ObjectProto) {
```

Comment:
thinking more about this... Object.prototype is reflective in the membrane, so target cannot be ObjectProto, we can remove this condition all together.

Reply:
Since these are handlers on both sides colors. I wonder if it could be the primary at some point (reflective).